### PR TITLE
refactor(general): minor cleanups in robustness framework 

### PR DESCRIPTION
**Pull request by julio-lopez**

### CODE REVIEW

**Comment by julio-lopez** [comment]
@chaitalisg

File: tests/robustness/robustness_test/main_test.go [hunk 1]
```diff
@@ -118,7 +118,7 @@ func (th *kopiaRobustnessTestHarness) init(ctx context.Context, dataRepoPath, me
 }
 
 func (th *kopiaRobustnessTestHarness) makeBaseDir() bool {
-	baseDir, err := os.MkdirTemp("", "engine-data-")
+	baseDir, err := os.MkdirTemp("", "robustness-")
```
Comment (adowair) [inline tests/robustness/robustness_test/main_test.go:121]: I am wary of this change for two reasons:

1. Could it cause confusion with the other robustness directories at this same (or lower) levels, if any exist?
2. Could it muck up some existing dependencies in codefresh (or other) scripts/configs.

**Review by adowair — COMMENTED** [review]
Looks good overall, thanks @julio-lopez !

File: tests/robustness/robustness_test/main_test.go (hunk 1 above)
Comment (julio-lopez) [reply tests/robustness/robustness_test/main_test.go:121]: This should be internal to a robustness job execution. The objective is to make it explicit and separate the data vs. the metadata directories, vs. the one used for other "temporary" data.

Nevertheless, I'll revert this change and we can merge it separately.

**Review by chaitalisg — COMMENTED** [review]
LGTM

**Pull request merged**